infrastructure/stacks/development-pipeline/codepipeline.tf

```diff
@@ -0,0 +1,106 @@
+resource "aws_codepipeline" "codepipeline" {
+  name     = var.development_pipeline_name
+  role_arn = aws_iam_role.codepipeline_role.arn
+
+  artifact_store {
+    location = var.codepipeline_artefact_bucket_name
+    type     = "S3"
+  }
+
+
+  stage {
+    name = "Source"
+    action {
+      category         = "Source"
+      name             = "Source"
+      owner            = "ThirdParty"
+      provider         = "GitHub"
+      version          = "1"
```

Comment:
Is there a particular version for this being version 1 and not version 2?

Reply:
Moved to version 2 which required us to change provider to CodeStarSourceConnection version 1 confusingly :)